net/tsaddr/tsaddr_test.go

```diff
@@ -222,3 +222,97 @@ func TestContainsExitRoute(t *testing.T) {
 		}
 	}
 }
+
+func TestIsTailscaleIPv4(t *testing.T) {
+	type args struct {
+		ip netip.Addr
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "TestTailscaleIPv4",
+			args: args{
+				ip: netip.MustParseAddr("100.67.19.57"),
+			},
+			want: true,
+		},
+		{
+			name: "TestNotTailscaleIPv4",
+			args: args{
+				ip: netip.MustParseAddr("10.10.10.10"),
+			},
+			want: false,
+		},
+		{
+			name: "TestTailscaleIPv6",
+			args: args{
+				ip: netip.MustParseAddr("fd7a:115c:a1e0:3f2b:7a1d:4e88:9c2b:7f01"),
+			},
+			want: false,
+		},
+		{
+			name: "TestNotTailscaleIPv6",
+			args: args{
+				ip: netip.MustParseAddr("bc9d:0aa0:1f0a:69ab:eb5c:28e0:5456:a518"),
+			},
+			want: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := IsTailscaleIPv4(tt.args.ip); got != tt.want {
+				t.Errorf("IsTailscaleIPv4() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestIsTailscaleIP(t *testing.T) {
+	type args struct {
+		ip netip.Addr
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
```

Comment:
the whole "args" type and even `name string` seems overkill here.

it's a func with one input that is stringable. It could be:


```go
	tests := []struct {
		in netip.Addr
		want bool
         }
...
      t.Errorf("IsTailscaleIP(%v) = %v, want %v", tt.in, got, tt.want)
```

Reply:
This is just my standard "generate tests for this method" boilerplate, I can change if you prefer the other way.